alpha/experience-platform-extension/set-up-the-sdk.md

```diff
@@ -88,27 +124,73 @@ ExperiencePlatformEvent event = new ExperiencePlatformEvent.Builder()
     .setXdmSchema(xdmData)
     .build();
 
-// send the event to the Experience Platform extension
+// Send the event to the Experience Platform extension
 ExperiencePlatform.sendEvent(event, null);
 ```
 
+{% endtab %}
+
+{% tab title="iOS" %}
 
+**Syntax**
+
+```java
+public static func sendEvent(experiencePlatformEvent: ExperiencePlatformEvent, responseHandler: ExperiencePlatformResponseHandler? = nil) 
+```
+
+- _experiencePlatformEvent (required)_ event to be sent to Adobe Data Platform. It should not be null.
+- _responseHandler (optional)_ callback to be invoked when the response handles are received from Adobe Data Platform. It can be called on a different thread and may be invoked multiple times.
+
+**Example (commerce event)** 
+
+```swift
+// Create the ExperiencePlatformEvent for your use case
+var productItem = ProductListItemsItem()
+productItem.name = "red ball"
+productItem.SKU  = "625-740"
+productItem.currencyCode = "USD"
+productItem.quantity     = 1
+productItem.priceTotal   = 9.95
+  
+var itemsList: [ProductListItemsItem]
+itemsList.append(productItem)  
+
+var commerce = Commerce()
+var productViews = ProductViews()
+productViews.value = 1
+commerce.productViews = productViews
+
+var xdmData = MobileSDKPlatformEventSchema()
+xdmData.eventType = "commerce.productListAdds"
+xdmData.commerce = commerce
+xdmData.productListItems = itemsList 
+
+// Send the event to the Experience Platform extension
+let event = ExperiencePlatformEvent(xdm:xdmData)
+let responseHandler = ResponseHandler()
+ACPExperiencePlatform.sendEvent(experiencePlatformEvent: event, responseHandler: responseHandler)
+```
+{% endtab %}
+{% endtabs %}
 
 ## Retrieving data from Adobe solutions
 
-As described in the [Sending events](#sending-events) section, `responseCallback` is an optional parameter. However, to be notified when a response is returned from the Adobe solutions, you can register a `responseCallback` that is invoked when new data is available from the server. This callback is called for each event handle that is returned by the server.
+As described in the [Sending events](#sending-events) section, the server-side event handle comes in chunks for the best performance. As a result, the `responseCallback` (Android) / `responseHandler` (iOS) is called multiple times. However, to be notified when a response is returned from the Adobe solutions, you can register a `responseCallback` that is invoked when new data is available from the server. This callback is called for each event handle that is returned by the server.
```

Comment:
Can we reword to something like:

`As described in the [Sending events](#sending-events) section, the server-side event handle comes in chunks for the best performance. However, to be notified when a response is returned from the Adobe solutions, you can register a `responseCallback` (Android) / `responseHandler` (iOS) that is invoked when new data is available from the server. This callback is invoked for each event handle that is returned by the server and so it can be called multiple times.`